apps/__init__.py

```diff
@@ -40,10 +40,33 @@ def register_extensions(app):
 
 def register_blueprints(app):
     for module_name in ('authentication', 'home', 'api', 'k8s', 'cli'):
-        module = import_module('apps.{}.routes'.format(module_name))
+        module = import_module(f'apps.{module_name}.routes')
         app.register_blueprint(module.blueprint)
-    # Load socketio endpoints
-    import_module("apps.events")
+
+    # (PoC) ContainerLabs: registra blueprint se habilitado
+    enable_clabs = app.config.get(
+        "ENABLE_CLABS",
+        str(os.getenv("ENABLE_CLABS", "false")).lower() in ("1", "true", "yes", "on")
+    )
+    if enable_clabs:
+        try:
+            clabs = import_module("apps.clabs.routes")
+            app.register_blueprint(clabs.blueprint)
+            app.logger.info("ContainerLabs blueprint registrado (ENABLE_CLABS=true).")
+        except Exception as e:
+            app.logger.error(f"Falha ao registrar ContainerLabs: {e}")
+    else:
+        app.logger.info("ContainerLabs desativado (ENABLE_CLABS=false).")
```

Comment:
I suggest that you try to generalize this with a configuration option like `OPTIONAL_MODULES` where you can include "clabs" by default and then only load the modules defined there. It would pretty much similar to what we have on lines 42-44, but looping over the `OPTIONAL_MODULES` list.